.nais/naiserator-prod-gcp.yaml

```diff
@@ -44,6 +44,8 @@ spec:
     outbound:
       external:
         - host: pdl-api.prod-fss-pub.nais.io
+        - host: dokdistfordeling-q1.prod-fss-pub.nais.io
```

Comment:
Er det riktig med `q1` her?

Reply:
@kaja-nav Ja, dokdistfordeling kjører fortsatt med det gamle ingress name schemet:
https://github.com/navikt/dokdistfordeling/blob/master/nais/q1-config.json#L37

Reply:
@torchr89 Men `q1` referer til et testmiljø, gjør det ikke? Dette er i prod-konfigen vår, og det er heller ikke samsvar med url som ligger i `prod-gcp.json`?

Reply:
@kaja-nav Ah, da blingset jeg visst helt! Trodde det bare var snakk om q1-miljø vs. dev

Reply:
Tror for øvrig man ikke trenger public ingress, siden det esyfovarsel bor i FSS, men kan heller bruke:
https://dokdistfordeling.intern.nav.no